core/state/statedb_arbitrum.go

```diff
@@ -39,31 +39,36 @@ var (
 	stylusEOFMagic       = byte(0xEF)
 	stylusEOFMagicSuffix = byte(0xF0)
 	stylusEOFVersion     = byte(0x00)
+	stylusEOFDictionary  = byte(0x00) // varies with dictionary selection
```

Comment:
I may be missing something, but this new variable is unused.

Reply:
Changed to just be a comment